openshift/4.0/templates/app/tenant-configMap.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: pims-app-tenant-config
```

Comment:
Separate ConfigMap template to update each environment through variables.  Makes it easier to update different environments.